SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  ViewController.swift
+//  SupportClient
+//
+//  Created by Omar Rasheed on 2/15/20.
+//  Copyright © 2020 Cornell Appdev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class ViewController: UIViewController {
+    
+    
+    var newConversationButton: UIButton!
+    var feedbackLabel: UILabel!
+    var searchController: UISearchController! // TODO: For later
+    var feedbackTableView: UITableView!
+        
+    var feedbackData: [Feedback]!
+    var filteredFeedbackData = [Feedback]() // TODO: For later
+    
+    var countEditTaps: Int = 0
+    var isFeedback: Bool!
+    let cellHeight: CGFloat = 75
+    let reuseID: String = "FeedbackTableViewCell"
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = Constants.backgroundColor
+        // Do any additional setup after loading the view.
+        
+        isFeedback = setUpData() == 0 ? false : true
```

Comment:
If we end up using the pod that I mentioned in a diff comment, then we can pretty much take this out entirely.